conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
+        try:
+            output_file = os.path.join(os.getcwd(), "graph.dot")
+            f = open(output_file, 'w')
+        except:
```

Comment:
except: then raise could be ommitted, it is already the default behavior, isn't it?